inference-engine/samples/common/samples/classification_results.h

```diff
@@ -74,6 +75,7 @@ class ClassificationResultT {
                               });
             for (unsigned j = 0; j < n; j++) {
                 output.at(i * n + j) = indexes.at(j);
+                _topK.push_back(indexes.at(j));
```

Comment:
The previous implementation would have worked for multiple batches, but it would have resulted in invalid results if the `print` method was called multiple times.

I have now refactored the changes so that it returns the same vector that was populated until now. Additionally, the computation part was moved to the constructor to avoid recreating it at each `print` call.